SiliconValleyOffice/GitHelp
# Add handling wikis from a Gitlab Group

**Description:**
The ability to handle wikis in GitHelp before only assumes that the current project is the GitHelp repo.
Now, we can clone and pull from a wiki of a GitLab group, and then update the current project's wiki with the Wiki from the GitLab group.


**Validation Steps**
- Switch to this version of Githelp
- Pick any project and use `ghCO` to clone that.
- Run `ghCW` to have a local copy of the current project's wiki
- Run `ghCWFGG <gitlab_group_name>` to clone the gitlab group's wiki
- Running `ghCWFGG <gitlab_group_name>` should return a message stating that `ghPWFGG` must be ran
- Try to make a change on the gitlab group's wiki then run `ghPWFGG <gitlab_group_name> `, this  pull in the change you just made
- Run `ghUWFGG <gitlab_group_name>` to push the GitLab group's wiki to your current project
